packages/vscode/src/index.ts

```diff
@@ -68,14 +87,14 @@ export async function activate(context: ExtensionContext) {
 
 	workspace.onDidChangeTextDocument((params: TextDocumentChangeEvent) => {
 		if (
-			['vue', 'svelte', 'javascript', 'typescript', 'javascriptreact', 'typescriptreact'].includes(
+			['vue', 'astro', 'javascript', 'typescript', 'javascriptreact', 'typescriptreact'].includes(
```

Comment:
Here and below, it's normal that it's Svelte and not Astro, changing this breaks getting changes for Svelte documents :smile: